e2e/ctl_v3_auth_test.go

```diff
@@ -960,3 +964,55 @@ func authTestEndpointHealth(cx ctlCtx) {
 		cx.t.Fatalf("endpointStatusTest ctlV3EndpointHealth error (%v)", err)
 	}
 }
+
+func authTestCertCNAndUsername(cx ctlCtx) {
```

Comment:
we should not use t.fatal all over the place.

fatal should be used for the pre-condition check, errorf should be used for the actual correctness checking.

Reply:
I see, thanks for pointing out. I'll update soon.